system/ee/ExpressionEngine/Library/Mime/MimeType.php

```diff
@@ -113,6 +113,10 @@ public function ofFile($path)
             $file_opening = file_get_contents($path, false, null, 0, 50);//get first 50 bytes off the file
             if (strpos($file_opening, 'RIFF') === 0 && strpos($file_opening, 'WEBPVP8') !== false) {
                 $mime = 'image/webp';
+                // PDF files start with "%PDF" (25 50 44 46) or " %PDF"
+                // @see https://en.wikipedia.org/wiki/Magic_number_%28programming%29#Examples
+            } else if (strpos($file_opening, '%PDF') >= 0) {
```

Comment:
In this case, `strpos` is going to evaluate as `false` if not found, and `0` if found at the start of the string.  Unfortunately, `false >= 0`  evaluates to true and so does `0 >= 0` so this if statement doesn't work. 

Can you change this to `!==  false`?

Reply:
I tested with different files. Sometime, the `%PDF` string appears at position 0 and sometimes at position 1.

Reply:
Yeah,  that's fine. The problem is that if no `%PDF` string appears it is still going to evaluate to true.